src/components/Tool/Appeal/AppealDetails/AppealHeaderInfo.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { Box, Grid, IconButton, Skeleton, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { AppealFieldsFragment } from 'pages/accountLists/[accountListId]/tools/GetAppeals.generated';
+import { EditIcon } from 'src/components/Contacts/ContactDetails/ContactDetailsTab/StyledComponents';
+import { useLocale } from 'src/hooks/useLocale';
+import { currencyFormat } from 'src/lib/intlFormat';
+import theme from 'src/theme';
+import AppealProgressBar from '../AppealProgressBar';
+
+export const appealHeaderInfoHeight = theme.spacing(9);
+
+const HeaderBarContactWrap = styled(Box)(() => ({
+  flex: 1,
+  display: 'flex',
+  alignItems: 'center',
+  gap: theme.spacing(2),
+  paddingTop: theme.spacing(1),
+  paddingBottom: theme.spacing(1),
+}));
+
+const AppealInfoHeader = styled(Typography)(() => ({
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const AppealInfo = styled(Typography)(() => ({
+  display: 'inline',
+  fontWeight: 'bold',
+}));
+
+const AppealInfoGrid = styled(Grid)(() => ({}));
```

Comment:
Are you going to add some styles in a later PR? If not, it might be best to just use `<Grid>`.